packages/apollo-vscode/README.md

```diff
@@ -0,0 +1,66 @@
+# Apollo VSCode
+
+An all-in-one tooling experience for developing apps with Apollo
+
+- Get instant feedback and intelligent autocomplete as you write queries
+- Run queries against your GraphQL server without leaving your editor
+- View performance statistics next to your query definitions
+
+![Code completing queries](images/variable-argument-completion.gif)
+![Running a query with variables](images/query-with-vars.gif)
+![Viewing Engine statistics](images/engine-stats.png)
+
+## Features
+
+- Loads your GraphQL schemas and queries automatically from an [Apollo Config](https://github.com/apollographql/apollo-cli/blob/master/packages/apollo-cli/README.md#configuration) file
+- Adds syntax highlighting for GraphQL files and `gql` templates inside JavaScript files
+- Code-completes fields, arguments, types, and variables in your queries
+- Lets you run queries, mutations, and subscriptions within the IDE with code-completion for variables
+- Displays performance statistics from [Apollo Engine](https://www.apollographql.com/engine) inline with your queries
+- Jump-to-definition for fragments and schema types
+- Detects fragment references and shows them next to definitions
+
+## How to get it?
+
+Open up VS Code and search for the extension "Apollo".
+
+## How to get it set up?
```

Comment:
What about uploading your schema to Engine? What if you're not running your GraphQL server locally?